bsp/boards/openmote-b/i2c.c

```diff
@@ -21,10 +21,10 @@
 
 #define I2C_PERIPHERAL          ( SYS_CTRL_PERIPH_I2C )
 #define I2C_BASE                ( GPIO_B_BASE )
-#define I2C_SCL                 ( GPIO_PIN_3 )
```

Comment:
why do you have to change I2C for OpenRadio work?!?

Reply:
Just using updated driver from the subghz project as is (in fact this setting breaks the slot debugpin)